src/vs/base/browser/ui/menu/menubar.ts

```diff
@@ -274,6 +274,10 @@ export class MenuBar extends Disposable {
 			}));
 
 			this._register(DOM.addDisposableListener(buttonElement, DOM.EventType.MOUSE_UP, (e) => {
+				if (e.defaultPrevented) {
+					return;
+				}
```

Comment:
@sbatten I tested this and it fixes the issue for level 1 menus but it doesn't fix it for submenus with level > 1
I think we need to add the same check here
https://github.com/microsoft/vscode/blob/26cb1c811c7c9d6da1490dfb65b4e4f695f86116/src/vs/base/browser/ui/menu/menu.ts#L392-L395
https://github.com/microsoft/vscode/blob/26cb1c811c7c9d6da1490dfb65b4e4f695f86116/src/vs/base/browser/ui/menu/menubar.ts#L371-L379